contracts/modules/Experimental/TransferManager/RestrictedPartialSaleTM.sol

```diff
@@ -0,0 +1,123 @@
+pragma solidity ^0.4.24;
+
+import "../../TransferManager/ITransferManager.sol";
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+
+/**
+ * @title This TransferManager is used to validate the transaction where partial balance of an 
+ * investor is not allowed to trnasfer beside investor is present in the exemption list
+ */
+contract RestrictedPartialSaleTM is ITransferManager {
+
+    // permission definition
+    bytes32 internal constant ADMIN = "ADMIN";
+
+    address[] public exemptAddresses;
+
+    mapping(address => uint256) public exemptIndex;
+
+
+    // Emit when the token holder is added/removed from the exemption list
+    event ChangedExemptWalletList(address indexed _wallet, bool _change);
+
+    /**
+     * @notice Constructor
+     * @param _securityToken Address of the security token
+     * @param _polyAddress Address of the polytoken
+     */
+    constructor (address _securityToken, address _polyAddress)
+    public
+    Module(_securityToken, _polyAddress)
+    {
+    }
+
+    /**
+    * @notice This function returns the signature of configure function
+    */
+    function getInitFunction() public pure returns (bytes4) {
+        return this.configure.selector;
+    }
+
+    /**
+     * @notice Used to initialize the variables of the contract
+     * @param _treasuryWallet Ethereum address of the treasury wallet
+     */
+    function configure(address _treasuryWallet) external onlyFactory {
+        _changeExemptionWalletList(_treasuryWallet, true);
+    }
+
+    /** 
+     * @notice Used to verify the transfer transaction and prevent a transfer if it passes the allowed amount of token holders
+     * @param _from Address of the sender
+     * @param _amount Amount to send
+     */
+    function verifyTransfer(
+        address _from,
+        address /* _to */,
+        uint256 _amount,
+        bytes /* _data */,
+        bool /* _isTransfer */
+    )
+        public
+        returns(Result)
+    {
+        if (!paused && _from != address(0) && exemptIndex[_from] == 0) {
+            return (ISecurityToken(securityToken).balanceOf(_from) == _amount ? Result.NA : Result.INVALID);
+        }
+        return Result.NA;
+    }
+
+    /**
+     * @notice Add/Remove wallet address from the exempt list
+     * @param _wallet Ethereum wallet/contract address that need to be exempted
+     * @param _change Boolean value used to add (i.e true) or remove (i.e false) from the list
+     */
+    function changeExemptWalletList(address _wallet, bool _change) external withPerm(ADMIN) {
```

Comment:
Please rename `_change` to something like `_exempted`. 
change as a name suggests that the current status should be changed (toggled).

Reply:
Yes, the current status should be changed. that is the whole idea. A switch to add or remove the addresses from the exemption list.

Reply:
As status must always be changed, that means change = true always.
However, that's not what change variable represents. It represents the value to which it should be changed rather than if it should be changed or not. As it represents the value, it should be called as the value.